wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsE2EIdentityCertificateView.swift

```diff
@@ -29,41 +29,41 @@ struct DeviceDetailsE2EIdentityCertificateView: View {
-                    statusText: viewModel.certificateStatus.titleForStatus(),
-                    textColor: SemanticColors.Label.textCertificateInvalid.swiftUIColor,
-                    image: Image(.certificateExpired)
-                )
-            case .revoked:
-                viewForStatus(
-                    titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
-                    statusText: viewModel.certificateStatus.titleForStatus(),
-                    textColor: SemanticColors.Label.textCertificateInvalid.swiftUIColor,
-                    image: Image(.certificateRevoked)
-                )
-            case .expired:
-                viewForStatus(
-                    titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
-                    statusText: viewModel.certificateStatus.titleForStatus(),
-                    textColor: SemanticColors.Label.textCertificateInvalid.swiftUIColor,
-                    image: Image(.certificateExpired)
-                )
-            case .valid:
-                viewForStatus(
-                    titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
-                    statusText: viewModel.certificateStatus.titleForStatus(),
-                    textColor: SemanticColors.Label.textCertificateValid.swiftUIColor,
-                    image: Image(.certificateValid)
-                )
-            case .none:
-                Spacer()
+            if let status = viewModel.e2eIdentityCertificate?.status {
+                switch status {
+                case .notActivated:
+                    viewForStatus(
+                        titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
+                        statusText: status.titleForStatus(),
+                        textColor: SemanticColors.Label.textCertificateInvalid.swiftUIColor,
+                        image: Image(.certificateExpired)
+                    )
+                case .revoked:
+                    viewForStatus(
+                        titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
+                        statusText: status.titleForStatus(),
+                        textColor: SemanticColors.Label.textCertificateInvalid.swiftUIColor,
+                        image: Image(.certificateRevoked)
+                    )
+                case .expired:
+                    viewForStatus(
+                        titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
+                        statusText: status.titleForStatus(),
+                        textColor: SemanticColors.Label.textCertificateInvalid.swiftUIColor,
+                        image: Image(.certificateExpired)
+                    )
+                case .valid:
+                    viewForStatus(
+                        titleText: L10n.Localizable.Device.Details.Section.E2ei.Status.title,
```

Comment:
What do you think should be the default value for the title since it is the same for all types of certificates?

Reply:
added a default value